services/web/server/src/simcore_service_webserver/storage_api.py

```diff
@@ -28,16 +33,59 @@ def _get_storage_client(app: web.Application) -> Tuple[ClientSession, URL]:
     return session, endpoint
 
 
+async def get_storage_locations(
+    app: web.Application, user_id: UserID
+) -> FileLocationArray:
+    log.debug("getting %s accessible locations...", f"{user_id=}")
+    session, api_endpoint = _get_storage_client(app)
+    locations_url = (api_endpoint / "locations").with_query(user_id=user_id)
+    async with session.get(f"{locations_url}") as response:
+        response.raise_for_status()
+        locations_enveloped = Envelope[FileLocationArray].parse_obj(
+            await response.json()
+        )
+        assert locations_enveloped.data  # nosec
+        log.info("%s can access %s", f"{user_id=}", f"{locations_enveloped.data=}")
+        return locations_enveloped.data
+
+
+async def get_project_total_size(
+    app: web.Application, user_id: UserID, project_uuid: ProjectID
+) -> ByteSize:
+    log.debug("getting %s total size for %s", f"{project_uuid=}", f"{user_id=}")
```

Comment:
We use a lot ``log.debug`` to trace start and finish of functions ... which is really handy! But we always rewrite it ... 

We have two great utils to support that in 

https://github.com/ITISFoundation/osparc-simcore/blob/94bbef366e9ad5f66f834f203211fc484db96d16/packages/service-library/src/servicelib/logging_utils.py

namely ``log_decorator`` or even a context manager (in case you want an smaller section inside the function) called ``log_context`` and ``log_catch``

Using those makes the code far cleaner, will produce uniform logs and is even faster to write :-)

Reply:
I agree but I do not see the result of the fct then